sw/device/tests/BUILD

```diff
@@ -1475,6 +1475,46 @@ opentitan_test(
     ],
 )
 
+opentitan_test(
+    name = "entropy_src_bypass_mode_health_test",
```

Comment:
Do we need to map this test to a SiVal test?

Reply:
I am not sure. @engdoreis if you can comment on this?

Reply:
yes, you have to link this test in the `bazel_target` field of the testplan, otherwise it will now reflect in the A1 Sheet.